### DISCUSSION TITLE
Sparkle Wikipedia page proposed for deletion

### DISCUSSION THREAD

**Discussion by CyberSkull** [sparkle-project/Sparkle#2529] [General]
The [Sparkle](https://en.wikipedia.org/wiki/Sparkle_(software)) Wikipedia page has been [proposed for deletion](https://en.wikipedia.org/wiki/Wikipedia:Proposed_deletion).

**Comment by zorgiepoo** [comment]
This page hasn't been significantly updated and doesn't include any notable citations/references which makes sense for why it's proposed for deletion.

Anyone here can feel free to contribute to this wiki page if they want. If someone wants to keep it alive they will need to find notable citations/references. Maybe just a list of notable applications is enough, not sure.

**Reply by zorgiepoo** [reply]
I may get to updating something on the page if nobody else gets to it before me

**Reply by zorgiepoo** [reply]
@CyberSkull So here's the thing. I, being a Sparkle maintainer, am not allowed to edit the page. If you don't want the page to be deleted I think you can object to it being removed and remove the proposed for deletion notice (if I read their guidelines right). If you don't do this, then it will probably be deleted and that's that (it doesn't have any substance currently anyway).

edit: if you want, I can provide a big list of notable secondary sources and such. But I can't write it.

**Comment by zorgiepoo** [comment]
Also taking a stab at updating Sparkle's own website for the list of apps https://github.com/sparkle-project/sparkle-project.github.io/pull/220 ...

**Reply by zorgiepoo** [reply]
@kornelski @tonyarnold @aonez @1024jp I'm using favicons from your websites/apps in this above change ^

**Reply by 1024jp** [reply]
@zorgiepoo Thank you for featuring my app! Moreover, thank you for maintaining the Sparkle framework always.

**Comment by CyberSkull** [comment]
I've updated the version number and the graphics on the page.

**Comment by CyberSkull** [comment]
@zorgiepoo The best thing to do as a major contributor to the project is to leave resources on the talk page for other editors to use. It is my understanding that that is allowed.

**Reply by zorgiepoo** [reply]
First let's have someone remove/defer the notability notice. Keeping the page alive is up to others
